beets/autotag/mb.py

```diff
@@ -607,6 +607,61 @@ def _parse_id(s: str) -> Optional[str]:
     return None
 
 
+# this was defined within the function below but pep8 made me move it here
+_trans_key = 'transl-tracklisting'
+_is_trans = lambda r: r['type'] == _trans_key and r['direction'] == "backward"
+
+
+def _find_actual_release_from_pseudo_release(pseudo_rel: Dict)\
+      -> Optional[Dict]:
+    relations = pseudo_rel['release']["release-relation-list"]
+
+    # currently we only support trans(liter)ation's
+    actual_id = next(filter(_is_trans, relations), {'target': None})['target']
+
+    if actual_id is None:
+        return None
```

Comment:
I think this could be a teensy bit simpler so it's easier to read/maintain. Something like this?

```py
translations = [r for r in relations if _is_trans(r)]
if not translations:
    return None
actual_id = translations[0]['target']
```